pkg/providers/pricing/pricing.go

```diff
@@ -215,7 +224,10 @@ func (p *Provider) UpdateOnDemandPricing(ctx context.Context, onDemandPrices map
 	p.onDemandPrices = lo.Assign(onDemandPrices)
 	p.onDemandUpdateTime = time.Now()
 	if p.cm.HasChanged("on-demand-prices", p.onDemandPrices) {
-		log.FromContext(ctx).WithValues("instance-type-count", len(p.onDemandPrices)).Info(fmt.Sprintf("updated on-demand pricing for region %s", p.region))
+		log.FromContext(ctx).WithValues(
+			"region", p.region,
+			"instanceTypeCount", len(p.onDemandPrices),
```

Comment:
I guess we are going with camelCasing for keys everywhere? Don't have a strong opinon, but let's make sure this is a concious choice.